web/static/js/phoenix_html.js

```diff
@@ -1,13 +1,20 @@
-function didHandleSubmitLinkClick(element){
+function isLinkToSubmitParent(element) {
+  var isLinkTag = element.tagName === 'A';
+  var shouldSubmitParent = element.getAttribute('data-submit') === 'parent';
+
+  return isLinkTag && shouldSubmitParent;
+}
+
+function didHandleSubmitLinkClick(element) {
   while(element) {
-    if(element.matches && element.matches('a[data-submit=parent]')){
+    if(isLinkToSubmitParent(element)) {
       var message = element.getAttribute('data-confirm');
       if (message === null || confirm(message)) {
-        element.parentNode.submit();
+        element.parentElement.submit();
```

Comment:
Also, it looks like `SVGElement` inherits from `Element`:

> The Element interface represents an object of a Document. This interface describes methods and properties common to all kinds of elements. Specific behaviors are described in interfaces which inherit from Element but add additional functionality. For example, the HTMLElement interface is the base interface for HTML elements, while the SVGElement interface is the basis for all SVG elements.

https://developer.mozilla.org/en-US/docs/Web/API/Element